upup/pkg/fi/cloudup/awstasks/load_balancer_attachment.go

```diff
@@ -24,21 +24,38 @@ import (
 	"github.com/golang/glog"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/cloudup/awsup"
+	"github.com/aws/aws-sdk-go/service/elb"
 )
 
 type LoadBalancerAttachment struct {
+	Name             *string
 	LoadBalancer     *LoadBalancer
+
+	// LoadBalancerAttachments now support ASGs or direct instances
 	AutoscalingGroup *AutoscalingGroup
-}
+	Subnet           *Subnet
 
-func (e *LoadBalancerAttachment) String() string {
-	return fi.TaskAsString(e)
+	// Here be dragons..
+	// This will *NOT* unmarshal.. for some reason this pointer is initiated as nil
+	// instead of a pointer to Instance with nil members..
+	Instance        *Instance
```

Comment:
We ended up not using the `Instance` member anyway after a comment from @photobox-services on bastion hosts being in an ASG instead of an Instance. So this code **could** in theory be removed in this PR..

So I coded in support for something, found a bug in it, we moved away from the original design, yet the bug remains and support has still be started. 

1. Do we keep the broken support?
2. Do we get this code out of this PR?